### PR TITLE
Feature/jobs radius

### PR DESCRIPTION
**Pull request by peggles2**
- SRCH-215
Expand the radius in requests to USAJOBS to 75 miles.

### CODE REVIEW

File: lib/jobs.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ module Jobs
     "|#{Date.current.year.to_s}"+
     "|#{BLOCKED_PHRASES}"+
     "|(#{SIMPLE_SINGULARS.join('|')})s?"
+  SEARCH_RADIUS = 75
```
Comment (MothOnMars) [inline lib/jobs.rb]: @peggles2 , you've set the radius as a constant, but you're not passing it to USAJOBS.

File: spec/lib/jobs_spec.rb [hunk 2]
```diff
@@ -34,6 +34,19 @@
         expect(search).to be_nil
       end
     end
+
+    context 'when the search is within the default radius, but in a different city' do
```
Comment (MothOnMars) [inline spec/lib/jobs_spec.rb]: @peggles2 , this doesn't necessarily test the radius, because this spec would pass if there were any jobs available in Baltimore. I would expect to see the `it 'searches USAJOBS with the correct params` spec updated with the expected radius.

**Comment by MothOnMars** [comment]
@peggles2 , there are a few RSpec failures to be addressed

File: lib/jobs.rb (hunk 1 above)
Comment (peggles2) [reply lib/jobs.rb]: Yes this is due to master getting merged. Not ready for code review yet. Thanks! :)

**Comment by peggles2** [comment]
Ready for review

File: spec/lib/jobs_spec.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@
     subject(:search) do
       Jobs.search({ query:'Nursing jobs',
                     organization_codes: 'HE38',
-                    location_name: 'Washington, DC, USA',
+                    location_name: 'Baltimore, MD, USA',
```
Comment (MothOnMars) [inline spec/lib/jobs_spec.rb]: Why the change from DC to Baltimore? This doesn't affect the test.

Comment (peggles2) [reply spec/lib/jobs_spec.rb]: This tests NIH search results to come back with results even if we are in Baltimore.
